scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -521,21 +519,40 @@ checkPeers() {
 check_peers="false"
 show_peers="false"
 selected_direction="out"
-data=$(curl -s -m ${EKG_TIMEOUT} -H 'Accept: application/json' "http://${EKG_HOST}:${EKG_PORT}/" 2>/dev/null)
-epochnum=$(jq '.cardano.node.ChainDB.metrics.epoch.int.val //0' <<< "${data}")
+if ! prom_port=$(jq -er '.hasPrometheus[1]' "${CONFIG}" 2>/dev/null); then prom_port=0 && USE_EKG='Y'; else USE_EKG='N'; fi
+if [[ ${USE_EKG} = 'Y' ]]; then
+  data=$(curl -s -m ${EKG_TIMEOUT} -H 'Accept: application/json' "http://${EKG_HOST}:${EKG_PORT}/" 2>/dev/null)
+  data_tsv=$(jq -r '[
+.cardano.node.ChainDB.metrics.epoch.int.val //0,
+.cardano.node.ChainDB.metrics.slotInEpoch.int.val //0,
+.cardano.node.ChainDB.metrics.slotNum.int.val //0,
+.cardano.node.Forge.metrics.remainingKESPeriods.int.val //0
+] | @tsv' <<< "${data}")
+  read -ra data_arr <<< ${data_tsv}
+  epochnum=${data_arr[0]}
+  slot_in_epoch=${data_arr[1]}
+  slotnum=${data_arr[2]}
+  remaining_kes_periods=${data_arr[3]}
+else
+  data=$(curl -s -m ${EKG_TIMEOUT} "http://${EKG_HOST}:${prom_port}/metrics" 2>/dev/null)
```

Comment:
should we check for `.hasPrometheus[0]` instead of EKG_HOST? I bet some folks may have changed Prometheus server listen port for grafana but not added one for EKG